config/rbac/service_account.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    app.kubernetes.io/name: serviceaccount
+    app.kubernetes.io/instance: manager-sa
+    app.kubernetes.io/component: rbac
+    app.kubernetes.io/created-by: nats-manager
+    app.kubernetes.io/part-of: nats-manager
+    app.kubernetes.io/managed-by: kustomize
+  name: manager
```

Comment:
`nats-manager`. Please make sure all the naming is consistent. There could be other projects with name `manager`. Better to use `nats-manager` everywhere relevant.

Reply:
it'll be nats-manager everywhere. kusomize prepends `nats-` prefix [here](https://github.com/muralov/nats-manager/blob/4a3bddd9c471f38707fc04aead992f2c129075ea/config/default/kustomization.yaml#L9)